### PR TITLE
Ra 466

### PR DESCRIPTION
**Pull request by ylatuya**

### CODE REVIEW

File: VAS.Tests/LocalStorage.cs [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using VAS.Core.Interfaces;
+using VAS.Core.Store;
+using VAS.Core.Filters;
+using VAS;
+
+namespace Tests
+{
+	public class LocalStorage : IStorage
```
Comment (aameijide) [inline VAS.Tests/LocalStorage.cs]: Document public methods

File: VAS.Tests/LocalStorage.cs [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+		{
+			return localStorage.ContainsKey (t.ID);
+		}
+
+		public int Count<T> () where T : IStorable
+		{
+			return localStorage.Count;
+		}
+
+		public StorageInfo Info {
```
Comment (aameijide) [inline VAS.Tests/LocalStorage.cs]: Properties must go before methods

File: VAS.Tests/LocalStorage.cs [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+		public bool Delete ()
+		{
+			return true;
+		}
+
+		#endregion
+	}
+
+
+	public class LocalDatabaseManager : IStorageManager
```
Comment (aameijide) [inline VAS.Tests/LocalStorage.cs]: I would split this in two different files

File: VAS.Tests/LocalStorage.cs (hunk 1 above)
Comment (ylatuya) [inline VAS.Tests/LocalStorage.cs]: The file is moved from longomatch, It's a dummy implementation for unit tests, it does not need documentation

File: VAS.Tests/LocalStorage.cs (hunk 2 above)
Comment (ylatuya) [inline VAS.Tests/LocalStorage.cs]: done

File: VAS.Tests/LocalStorage.cs (hunk 3 above)
Comment (ylatuya) [inline VAS.Tests/LocalStorage.cs]: done

**Pull request merged**